server/resource/allUsers.py

```diff
@@ -0,0 +1,15 @@
+from flask_restful import Resource
+from flask_jwt_extended import (jwt_required)
+import sys
+sys.path.append("..")
+from models import UserModel
+
+
+class AllUsers(Resource):
+    @jwt_required
+    def get(self):
+        return UserModel.return_all()
+
+    @jwt_required
+    def delete(self):
+        return UserModel.delete_all()
```

Comment:
We probably don't want to have these routes tbh, because that means any user could get all users and delete all of them. I'm assuming you added these for testing purposes but can be removed before you merge this. 

To do these types of actions, it might be easier to use a SQL viewer like (Sequel Pro, MySQL Workbench, etc.)

As another FYI, the file naming convention in python is snake_case (so change that for all of your resources)